### PR TITLE
Fix benchmark build

### PR DESCRIPTION
**Pull request by enthusiastmartin**
Issue introduced when new payment pallet was introduced. 

Only present when building node for benchmarking.

### CODE REVIEW

File: node/Cargo.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ module-amm-rpc = { path = '../pallets/amm/rpc', version = '2.0.0' }
 
 # Substrate dependencies
 frame-benchmarking = '2.0.0'
-frame-benchmarking-cli = '2.0.0'
+frame-benchmarking-cli = {version = '2.0.0', optional = true }
```
Comment (jak-pan) [inline node/Cargo.toml:33]: Is this formatted?

Comment (enthusiastmartin) [reply node/Cargo.toml]: done. looks better, consistent now.

Comment (jak-pan) [reply node/Cargo.toml]: Was it actually formatted or did you fix this manually? 

Maybe we just need to set some settings. I've noticed similar stuff in the code a few times.

**Review by jak-pan — APPROVED** [review]

File: node/Cargo.toml (hunk 1 above)
Comment (enthusiastmartin) [reply node/Cargo.toml]: I did it manually. Is there something which can format these type of files ? i guess there is but apparently i dont use it yet.

**Pull request merged**